src/tokens/letterSpacing.js

```diff
@@ -1,14 +0,0 @@
-const light = (core) => ({
```

Comment:
It looks like the other files were renamed, but this one is showing as deleted. Is that just a git code diff bug?

Reply:
I think this was just a diff bug, but I'm removing the TS version anyway now since it's no longer needed.